CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+1.5.0 (2020-02)
```

Comment:
Why so many changes?

Reply:
Changelog hasn't been updated in a while, needed all these to be included.